src/ralph/helpers.py

```diff
@@ -48,16 +48,23 @@ def _cache_key_hash(func, *args, **kwargs):
     return pickle.dumps((func.__module__, func.__name__, args, kwargs))
 
 
-def cache(seconds=300, cache_name=DEFAULT_CACHE_ALIAS):
+def cache(seconds=300, cache_name=DEFAULT_CACHE_ALIAS, skip_first=False):
     """
     Cache the result of a function call with particular parameters for specified
     number of seconds.
+
+    Args:
+        * skip_first - set to True if first argument should not be considered
```

Comment:
Is this a big change to make seperate fns, one for skipping self (the new one) and old one?